IBPSA/package.mo

```diff
@@ -165,6 +174,9 @@ This section summarizes the releases of the <code>IBPSA</code> library.
 </p>
 <ul>
 <li>
+<a href=\"modelica://IBPSA.UsersGuide.ReleaseNotes.Version_4_0_0\">Version 4.0.0</a>(xxx)
```

Comment:
Why not set a date?

Reply:
Because the date will be set when there is an official release that then will be tagged.